### ISSUE TITLE
just-your-type

### ISSUE THREAD

**Issue by VanshitaGandhi** [VanshitaGandhi/just-your-type#1]
svg color 
@mackenzierose

**Comment by VanshitaGandhi** [comment]
I chose this quote as I'm fascinated by poetry. I like this poetry by Ernest Hemingway as he is a very honest poet and that reflects in his body of work. Since the quote is a bit dark, many people perceive it as negative. But in reality, it's just a beautiful  acknowledgment of strength and perseverance. Hence, I chose _Petit Formal Script Font_ as it flows beautifully. Also, this font is used for the words _you_ and _suffering_ to create emphasis. I chose _Montserrat_, a sans-serif to balance the design and keep it simple. My chosen type scale perfectly fit all the screen-sizes. To make the website responsive, I chose this type scale. I chose the background as a gradient that goes from deep blue to black indicating the depth of the poetry. I chose white for the font to keep its simplicity and create a clear contrast to enhance readability. Although the grids weren't necessary for my design, I still used them to keep the code clean and responsive. I used Gridifier, Modulifier and Typografier to practice what I've learnt so far and to eliminate redundant code. The main challenge I faced was to place things correctly in a grid at different screen-sizes and I overcame it by using position absolute and position relative in media queries. 
@muxlowc
